data/fieldlist_NCAR.jsonc

```diff
@@ -0,0 +1,216 @@
+      "scalar_coord_templates": {"plev": "V{value}"},
+      "ndim": 4
+    },
+    "Z3": {
+      "standard_name": "geopotential_height",
+      "units": "m",
+      // note: 4d name is 'Z3' but Z500 = height at 500 mb, etc.
+      "scalar_coord_templates": {"plev": "Z{value}"},
+      "ndim": 4
+    },
+    "Q": {
+      "standard_name": "specific_humidity",
+      "units": "1",
+      "ndim": 4
+    },
+    "OMEGA": {
+      "standard_name": "lagrangian_tendency_of_air_pressure",
+      "units": "Pa s-1",
+      "scalar_coord_templates": {"plev": "OMEGA{value}"},
+      "ndim": 4
+    },
+    "TS": {
+      "standard_name": "surface_temperature",
+      "units": "K",
+      "ndim": 3
+    },
+    "PS": {
+      "standard_name": "surface_air_pressure",
+      // note: not hPa
+      "units": "Pa",
+      "ndim": 3
+    },
+    "PRECT": {
+      "standard_name": "precipitation_rate",
+      "units": "m s-1",
+      "ndim": 3
+    },
+    "PRECC": {
+      "standard_name": "convective_precipitation_rate",
+      "units": "m s-1",
+      "ndim": 3
+    },
+    "TREFHT" : {
+      // correct name? CMIP6 equivalent should be tas, temp at 2m ref height
+      "standard_name": "air_temperature",
+      "units": "K",
+      "ndim": 3
+    },
+    "TAUX": {
+      "standard_name": "surface_downward_eastward_stress",
+      "units": "Pa",
+      "ndim": 3
+    },
+    "TAUY": {
+      "standard_name": "surface_downward_northward_stress",
+      "units": "Pa",
+      "ndim": 3
+    },
+    "PSL": {
+      "standard_name": "air_pressure_at_mean_sea_level",
+      "units": "Pa",
+      "ndim": 3
+    },
+    // radiative fluxes:
+    // "rsus": {
+    //   "standard_name": "surface_upwelling_shortwave_flux_in_air",
+    //   "units": "W m-2",
+    //   "ndim": 3
+    // },
+    // "rsds": {
+    //   "standard_name": "surface_downwelling_shortwave_flux_in_air",
+    //   "units": "W m-2",
+    //   "ndim": 3
+    // },
+    // "rsdt": {
+    //   "standard_name": "toa_incoming_shortwave_flux",
+    //   "units": "W m-2",
+    //   "ndim": 3
+    // },
+    // "rsut": {
+    //   "standard_name": "toa_outgoing_shortwave_flux",
+    //   "units": "W m-2",
+    //   "ndim": 3
+    // },
+    "FLUS": {
+      // correct name? Guessed according to pattern -- no FLUS, only the net combination?
+      "standard_name": "surface_upwelling_longwave_flux_in_air",
+      "units": "W m-2",
+      "ndim": 3
+    },
+    "FLDS": {
+      "standard_name": "surface_downwelling_longwave_flux_in_air",
+      "units": "W m-2",
+      "ndim": 3
+    },
+    "FLUT": {
+      "standard_name": "toa_outgoing_longwave_flux",
+      "units": "W m-2",
+      "ndim": 3
+    },
+    "SHFLX": {
+      "standard_name": "surface_upward_sensible_heat_flux",
+      "units": "W m-2",
+      "ndim": 3
+    },
+    "LHFLX": {
+      "standard_name": "surface_upward_latent_heat_flux",
+      "units": "W m-2",
+      "ndim": 3
+    },
+    // Variables for AMOC_3D_Structure module:
```

Comment:
Yes: some (but not all) are variables used by Gen 2 PODs. To reduce the need for manual maintenance of these `fieldlist_*` files, I'd like to dump the full list of recognized native variable names for each convention, rather than adding them as needed.

Actually *finding* these variable names has been surprisingly difficult; NCAR provides [this list](https://www.cesm.ucar.edu/models/cesm2/atmosphere/docs/ug6/hist_flds_f2000.html) for their atmosphere model only. The closest thing I've been able to find at GFDL are [these tables](https://nomads.gfdl.noaa.gov/CM2.X/CM2.0/available_data.html) for CM2.0, but they omit variable units. If you know of a better source for this info, please tell me!